.gitignore

```diff
@@ -16,6 +16,9 @@ __pycache__/
 # VS Code
 .vscode
 
+# YAML generated from running example kfp.compile scripts
+examples/**/*.yaml
```

Comment:
This seems like an overly restrictive ignore. I can definitely imagine future examples that we'd want to have include yaml files. Can we make this more specific to just the kfp compilation files?

Reply:
Roger that, i will scope it to just `examples/llmcompressor/`